backend/.env.example

```diff
@@ -11,3 +11,8 @@ DB_CONNECTION_STRING=[db_client]://[db_username]:[db_user_password]@localhost:[d
 DB_DIALECT=pg
 DB_POOL_MIN=2
 DB_POOL_MAX=10
+
+#
+# TOKEN
+#
+SECRET_KEY=secret
```

Comment:
I suggest naming it as `JWT_SECRET_KEY`, I think current name doesn't give much context on what is this variable related to

Reply:
It has been named ACCESS_TOKEN_SECRET at the fist time but @andanf-e suggested to rename it to SECRET_KEY.
so what do you think it would be better? @andanf-e @artemmatiushenko1 
It seems good for me because it is in Token section.

Reply:
I already have it called SECRET_KEY, but that's because I have Irina's branch. JWT_SECRET_KEY is also good. Who knows how many more SECRET_KEYs there will be.

Reply:
Let's make it JWT_SECRET_KEY, yep